### DISCUSSION TITLE
I want to add a button on my website. When clicked, it should open the GrapesJS editor, and my template should also be loaded into GrapesJS so that I can edit it.

### DISCUSSION THREAD

**Discussion by ZaheerNiazi007** [GrapesJS/grapesjs#5874] [Q&A]
![Screenshot 2024-05-08 122455](https://github.com/GrapesJS/grapesjs/assets/72734684/9b655a28-6c13-40fd-80d9-1d8bb57a3671)
When click on Open Editor this template will open in GrapesJS

**Comment by mosh-tudor** [comment]
@ZaheerNiazi007 

What's your question?

According to the description, everything seems simple and GrapesJs has all the necessary functionality out of the box.

**Reply by ZaheerNiazi007** [reply]
@mosh-tudor To transfer all the HTML and CSS content from a simple application page into the GrapesJS canvas, you can implement a functionality where clicking on a button retrieves the data from the template and pastes it onto the GrapesJS canvas. This enables you to modify and customize the content within GrapesJS.

**Reply by mosh-tudor** [reply]
Nice. So what's your question?

**Reply by ZaheerNiazi007** [reply]
@mosh-tudor that is my question how to acheive this?

**Comment by artf** [comment]
You usually start by reading the documentation, trying to solve your problem, and then you can ask a question if something doesn't work as expected.

**Reply by ZaheerNiazi007** [reply]
Sure, thanks. I'm already working on it. It's a very wonderful library, thanks
